tweaks.py

```diff
@@ -720,6 +727,31 @@ def get_indefinite_article(string):
   else:
     return "a"
 
+def add_article_before_item_name(item_name):
+  # Adds a grammatical article ("a", "an", "the", or nothing) in front an item's name.
+  article = None
+  if re.search(r"\d$", item_name):
+    article = None
+  elif (PROGRESS_ITEMS + NONPROGRESS_ITEMS).count(item_name) > 1:
+    article = get_indefinite_article(item_name)
+  elif item_name in CONSUMABLE_ITEMS:
+    article = get_indefinite_article(item_name)
+  elif item_name in DUPLICATABLE_CONSUMABLE_ITEMS:
+    article = get_indefinite_article(item_name)
+  elif item_name.lower().endswith(" small key"):
+    article = get_indefinite_article(item_name)
+  elif item_name in ["Beedle's Chart", "Tingle's Chart", "Maggie's Letter"]:
+    article = None
+  elif item_name.endswith("'s Pearl"):
+    article = None
+  elif item_name in ["Father's Letter", "Moblin's Letter"]:
+    article = get_indefinite_article(item_name)
```

Comment:
Would it make sense to have these be consistent with the textboxes that show up when you get the items?

![Screenshot_20221221_004252_Twitch](https://user-images.githubusercontent.com/8262173/208830897-38b2d8fe-c383-4f73-843a-6e104b8e255d.png)

![Screenshot_20221221_004557_YouTube](https://user-images.githubusercontent.com/8262173/208830914-a1b784e7-671c-44a9-aef5-53e75b090c6d.png)